packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -541,6 +549,12 @@ export const RemoteServerFactory = {
         app.use(ssrRequestProcessorMiddleware)
     },
 
+    _setRequestId(res, req) {
+        const requestContext = req.apiGateway.event.requestContext
+        const locals = res.locals
+        locals.requestId = requestContext.requestId
+    },
```

Comment:
@alexvuong @bendvc Ya wanna chat about this? I feel like `locals` has been a mistake 90% of the time. Why do you need this attached to locals, with all the other work that is going on? I though you were going to have a `useRequest` hook or something – that'd give you access to the request everywhere, no?

Reply:
Happy to chat.. I believe you are mistaking this PR for another